testing/junit5-suite/src/main/java/io/helidon/testing/junit5/suite/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * jUnit test suite.
```

Comment:
I don't understand what this is. JUnit already [has the concept of a test suite](https://docs.junit.org/current/user-guide/#junit-platform-suite-engine). Also, why is this part of the Helidon Data PR?